distribution/pom.xml

```diff
@@ -164,6 +164,10 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Is there a reason why you enable it per project? There are projects where it doesn't make sense?